flink-formats/flink-json/src/main/java/org/apache/flink/formats/json/canal/CanalJsonFormatFactory.java

```diff
@@ -43,6 +43,7 @@
 import java.util.HashSet;
 import java.util.Set;
 
+import static org.apache.flink.formats.json.JsonFormatOptions.ALLOW_NON_NUMERIC_NUMBERS;
```

Comment:
Canal / Maxwell / Debezium shouldn't be importing from `JsonFormatOptions`, those options should be redirected through `CanalJsonFormatOptions` etc. This is already wrong for `ENCODE_DECIMAL_AS_PLAIN_NUMBER`. Can you please fix these (and in the other locations)?

Reply:
Thansk.  I will fix later.